react/src/hooks/users/useUserQuery.tsx

```diff
@@ -12,7 +12,9 @@ async function fetchUser(username: string) {
  * That is, return all details about a particular user,
  * including MinIO credentials.
  */
-export function useUserQuery(username: string) {
-    const result = useQuery<User, Response>(["users", username], () => fetchUser(username));
+export function useUserQuery(username: string, options: UseQueryOptions<User, Response> = {}) {
+    const result = useQuery<User, Response>(["users", username], () => fetchUser(username), {
+        ...options,
+    });
```

Comment:
You should be able to pass `options` through, no need for a spread copy.

Reply:
I am passing `options` here because `useUserQuery` is called in `Settings/index.tsx` and `UserDetails.tsx`. I would like to handle the error in `Settings/index.tsx` but not in `UserDetails.tsx` because `useUsersQuery` is already called in `UserDetails.tsx`. I am handling network error caused by `useUsersQuery`. 

What do you think about this?

Reply:
I mean that you should be able to directly pass in `options` as an argument. You shouldn't need to make a copy with `{ ...options }` as you aren't modifying any properties.

Reply:
Sorry, I get what you mean.